build.gradle.kts

```diff
@@ -60,29 +60,13 @@ metadata {
 
 /* ******************** dependencies ******************** */
 
-val internalPlatform by configurations.creating {
-    isVisible = false
-    isCanBeConsumed = false
-    isCanBeResolved = false
-}
-
-configurations {
-    compileClasspath.get().extendsFrom(internalPlatform)
-    runtimeClasspath.get().extendsFrom(internalPlatform)
-    testCompileClasspath.get().extendsFrom(internalPlatform)
-    testRuntimeClasspath.get().extendsFrom(internalPlatform)
-    javadocLinks.get().shouldResolveConsistentlyWith(runtimeClasspath.get())
-}
-
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    internalPlatform(platform("com.hivemq:hivemq-main-platform"))
-
-    api("com.hivemq:hivemq-extension-sdk:${version}")
-    api("org.slf4j:slf4j-api")
+    api(libs.hivemq.extensionSdk)
```

Comment:
```suggestion
    api(libs.hivemq.extensionSdk) {
        version {
            require(project.version.toString())
        }
    }
```

Reply:
There is a better alternative. I will update this line.